groclient/lib.py

```diff
@@ -606,70 +606,63 @@ def get_geojson(access_token, api_host, region_id, zoom_level):
 
 
 def get_ancestor(access_token, api_host, entity_type, entity_id, distance=None,
-                   include_details=True):
+                 include_details=True, ancestor_level=None, include_historical=True,):
     url = '/'.join(['https:', '', api_host, 'v2/{}/belongs-to'.format(entity_type)])
     headers = {'authorization': 'Bearer ' + access_token}
     params = {'ids': [entity_id]}
     if distance:
         params['distance'] = distance
+    elif ancestor_level:
```

Comment:
ancestor_level only applies to regions, so we should add another condition to check for entity type, otherwise the request will fail

Reply:
Ok definitely, will do, thank you!